### DISCUSSION TITLE
Allow disabling just "E" and/or don't auto create fork

### DISCUSSION THREAD

**Discussion by brandonchinn178** [community/community#40730] [Repositories]
### Select Topic Area

Product Feedback

### Body

If I'm browsing someone else's repo and accidentally click "E", GitHub will automatically fork the repo and put me in the editor view. Even if I immediately cancel, I then have to go delete my fork of the repo.

Possible solutions:
1. Allow disabling specific keyboard shortcuts (instead of one toggle for all)
2. Only enable edit for repos you already have write access on
3. Open a dialog to confirm creating a fork if one doesn't already exist
4. Only create the fork when saving the edits
    * While we're at it, it would be nice to specify the fork name and branch name when making a commit in the web editor, instead of automatically using the same repo name and `patch-1` branch

**Comment by ossanna16** [comment]
Hi there @brandonchinn178 and welcome to our community! Thank you for the feedback!
